ads/torimochi.md

```diff
@@ -0,0 +1,39 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Teads
+
+## Example
+
+```html
+<amp-ad width="300" height="torimochi"
+    type="torimochi"
+    data-type="banner"
+    data-area="300x250">
+</amp-ad>
+```
+
+## Configuration
+
+For details on the configuration semantics, please contact the ad network or refer to their documentation. 
+
+Supported parameters:
+
+- `data-adtype`
```

Comment:
Thanks for adding your documentation. For the supplied parameters, could you specify which are optional vs. required? [Here is an example](https://github.com/ampproject/amphtml/blob/master/ads/_ping_.md ). Thank you! 😄